build/storage/tests/ptf_scripts/test_setup.py

```diff
@@ -0,0 +1,150 @@
+import sys
+
+sys.path.append('../')
+
+from python_system_tools.terminal import SSHTerminal
+from python_system_tools.test_platform import TestPlatform
+from python_system_tools.config import ipu_storage_address, storage_address, user_name, password
+from ptf.base_tests import BaseTest
+
+
+class TestOS(BaseTest):
+
+    def setUp(self):
+        self.setup_proxy = TestPlatform(SSHTerminal(address=ipu_storage_address, user=user_name, password=password))
+        self.setup_storage = TestPlatform(SSHTerminal(address=storage_address, user=user_name, password=password))
+
+    def runTest(self):
+        assert self.setup_storage.os in ("Fedora", "Ubuntu")
+        assert self.setup_proxy.os in ("Fedora", "Ubuntu")
+
+    def tearDown(self):
+        pass
+
+
+class TestPM(BaseTest):
+
+    def setUp(self):
+        self.setup_proxy = TestPlatform(SSHTerminal(address=ipu_storage_address, user=user_name, password=password))
+        self.setup_storage = TestPlatform(SSHTerminal(address=storage_address, user=user_name, password=password))
+
+    def runTest(self):
+        assert self.setup_storage.pm in ("dnf", "apt")
+        assert self.setup_proxy.pm in ("dnf", "apt")
+
+    def tearDown(self):
+        pass
+
+
+class TestCheckVirtualization(BaseTest):
+
+    def setUp(self):
+        self.setup_proxy = TestPlatform(SSHTerminal(address=ipu_storage_address, user=user_name, password=password))
+        self.setup_storage = TestPlatform(SSHTerminal(address=storage_address, user=user_name, password=password))
+
+    def runTest(self):
+        return_values = TestPlatform.setup_on_both_machines(
+            self.setup_storage.check_virtualization,
+            self.setup_proxy.check_virtualization,
+            timeout=30,
+        )
+
+        assert return_values == (True, True)
+
+    def tearDown(self):
+        pass
+
+
+class TestCheckKVM(BaseTest):
```

Comment:
This test will be applicable only for KVM case host-target platform

Reply:
Removed